src/test/resources/filter-tests.adoc

```diff
@@ -1837,37 +1837,39 @@ RETURN graphql.labels(company) AS _labels,
 
 === Filter as variable
 
-CAUTION: Is not working
-
 .GraphQL-Query
 [source,graphql]
```

Comment:
do we also have a test where inside of a filter a variable/parameter is used?

Reply:
Do you mean the `Variable pass through`-Test?